test/services/return-requirements/delete-note.service.test.js

```diff
@@ -0,0 +1,64 @@
+// write test in hapi/yar for delete-note.service.js
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+const { describe, it, beforeEach } = exports.lab = Lab.script()
```

Comment:
NIT - adding some whitespace

```suggestion
const Sinon = require('sinon')

const { describe, it, beforeEach } = exports.lab = Lab.script()
```